### DISCUSSION TITLE
Embedding model, do I need to set additional token? Why does my ragflow-server keep reporting errors in its ERROR.log?

### DISCUSSION THREAD

**Discussion by dpnotnull** [infiniflow/ragflow#1941] [Q&A]
Login in the container ragflow-server, the ERROR.log within /ragflow/logs/api

keep reporting:

Can't update token usage for 490c8ecc561c11efab590242ac140006/EMBEDDING
Can't update token usage for 490c8ecc561c11efab590242ac140006/EMBEDDING
Can't update token usage for 490c8ecc561c11efab590242ac140006/EMBEDDING
Can't update token usage for 490c8ecc561c11efab590242ac140006/EMBEDDING
Can't update token usage for 490c8ecc561c11efab590242ac140006/EMBEDDING
......

**Comment by dpnotnull** [comment]
And Embedding models have been tried for both: BAAI/bge-large-zh-v1.5 & sentence-transformers/all-MiniLM-L6-v2

Can you give me some hints or suggestions for further troubleshooting the problem. Many thanks～

**Reply by hangters** [reply]
hi @dpnotnull   This is because the default embedding model was not added to the relevant data table, but this does not affect usage.

**Reply by dpnotnull** [reply]
Got it, many thanks~
